### DISCUSSION TITLE
DS3 Controller random inputs

### DISCUSSION THREAD

**Discussion by Vendeleska** [nefarius/DsHidMini#300] [Q&A]
Hey, I have been using this app for a while but lately every time I start my PC and connect my DS3 on Bluetooth, it goes crazy on random inputs. I used to use it wired with an older Virtual emulator that for the life of me I cant remember the name of and it was working fine but ever since past a couple of months using it on this app wirelessly its been going crazy. Is this a hardware issue?

**Comment by nefarius** [comment]
DsHidMini doesn't randomly decide to go crazy and invent its own inputs.
